### PR TITLE
Updates for numerals and Odia

### PR DESCRIPTION
**Pull request by jowilco**

### CODE REVIEW

**Comment by acrolinxatmsft1** [comment]
## Acrolinx Scorecards

**A minimum total score of 80 is required.**
 
Select the total score link to review all feedback on clarity, consistency, tone, brand, terms, spelling, grammar, readability, and inclusive language. _You should fix all spelling errors regardless of your total score_. Fixing spelling errors helps maintain customer trust in overall content quality.

| Article | Total score<br>(Required: 80) | Words + phrases<br>(Brand, terms) | Correctness<br>(Spelling, grammar) | Clarity<br>(Readability) |
|---------|:--------------:|:--------------------:|:------:|:---------:|
| [:white_check_mark:](# "Successfully processed") [globalization/input/odia-ime.md](https://github.com/MicrosoftDocs/globalization/blob/a440d9f0b5ea24e43f59992e07523ea03613494f/globalization%2Finput%2Fodia-ime.md) | [79](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/255ca10e-2481-412b-900b-01e3ded394eb) | 100 | 16 | 82 |
| [:white_check_mark:](# "Successfully processed") [globalization/locale/numeric-notation.md](https://github.com/MicrosoftDocs/globalization/blob/a440d9f0b5ea24e43f59992e07523ea03613494f/globalization%2Flocale%2Fnumeric-notation.md) | [81](https://microsoft-ce-csi.acrolinx.cloud/api/v1/checking/scorecards/f55a98d4-be98-4cd6-b8cb-73c7b2b9a027) | 100 | 35 | 74 |

**More information about Acrolinx**

- [Install Acrolinx locally for VSCode](https://review.docs.microsoft.com/help/contribute/contribute-acrolinx-vscode)
- [False positives or issues](https://aka.ms/acrolinxbug)
- [Troubleshooting issues with Acrolinx](https://review.docs.microsoft.com/help/contribute/acrolinx-error-messages)

**Review by joeyobr — APPROVED** [review]
Looks good to me.

File: globalization/locale/numeric-notation.md [hunk 1]
```diff
@@ -29,21 +29,18 @@ Several other sets of digits are used:
...
 | Kannada | ೦ ೧ ೨ ೩ ೪ ೫ ೬ ೭ ೮ ೯ |
 | Khmer | ០ ១ ២ ៣ ៤ ៥ ៦ ៧ ៨ ៩ |
-| Korean (한자어 숫자/Sino-Korean numbers) | 일 이 삼 사 오 육 칠 팔 구… |
-| Korean (고유어 숫자/native Korean numbers) | 하나 둘 셋 넷 다섯 여섯 일곱 여덟 아홉… |
 | Lao | ໐ ໑ ໒ ໓ ໔ ໕ ໖ ໗ ໘ ໙ |
 | Malayalam | ൦ ൧ ൨ ൩ ൪ ൫ ൬ ൭ ൮ ൯ |
 | Mon–Burmese | ၀ ၁ ၂ ၃ ၄ ၅ ၆ ၇ ၈ ၉ |
 | New Tai Lue | ᧐ ᧑ ᧒ ᧓ ᧔ ᧕ ᧖ ᧗ ᧘ ᧙ |
 | N’ko | &#x2067;߀ ߁ ߂ ߃ ߄ ߅ ߆ ߇ ߈ ߉ &#x2069;|
 | Odia | ୦ ୧ ୨ ୩ ୪ ୫ ୬ ୭ ୮ ୯ |
```
Comment (joeyobr) [inline globalization/locale/numeric-notation.md:37]: Maybe change this to "Odia (Oriya)"?

**Review by joeyobr — COMMENTED** [review]
Otherwise, looks good.

**Pull request merged**